modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather log data about a failed Agent-based installation to provide for a support case.
+
+.Procedure
+
+. Run the following command and collect its output:
```

Comment:
I think the `its` part needs a bit more description. Maybe like 
`Collect the output of a failed Agent-based installation`
^^
This might not be technically correct but I think you get what I mean 😄 

Also, not a huge deal, but usually we'd want to write the description of the command, then add the "by running to by entering the following command" part
[Shown mostly here in the doc guidelines  ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L1048)

Reply:
Regarding "its" needing more description: "it" refers to the command, so the instruction is to run the command and copy whatever output it gives you. I could reword it to something like "Run the following command and collect the output of the command" but imo I think it's a bit too wordy.

Regarding the "by running the following command" structure: The guideline says that structure should be followed in most cases, however in these steps, running the command is the first of two actions taken in the step (running the command and then collecting the output of the command). I'd have to split the step into two steps to follow what's really a guideline, whereas this format still follows the requirement to make sure there's an explicit instruction to "run" the command.

Is it fine to keep this step (and the others like it) the way it is? Let me know what you think!

Reply:
Ohhh okay! I wasn't correctly reading what I thought the command was doing. And yeah that makes sense for the format of the procedure referring to the "by running" part. I'm still wondering, what do you think of just, 
`Run the following command and collect the output`
I personally feel like this ^^ is a bit simpler and conveys the same thing. But thats also a bit nitty so its not a huge deal if you want to keep what you have 😄

Reply:
Sure thing, that wording works for me